docs/references/vod/patch.mdx

```diff
@@ -75,7 +75,7 @@ curl -X PATCH https://livepeer.studio/api/asset/{id} \
 -d {  "meta": {
         "title": "My awesome video",
         "description": "This is a video of my awesome life",
-        "tags": ["awesome", "life", "video"],
+        "tags": "awesome","life","video"
```

Comment:
This is still not valid JSON. You can any online tool to validate these. If you throw that on https://jsoneditoronline.org/ you'll get the errors.

As I've mentioned, this has to be a single string, not multiple strings separated by commas. That's not a valid JSON unless this field was an array.